### PR TITLE
add conclusion docs

### LINKED ISSUE #241: Conclusion or Summary doc [meteor]
The tutorial should include a summary/conclusion/congratulations page that comes after `monitor-model.md` letting the user know they've completed all the steps in the tutorial and re-iterate what they've accomplished with all of the AICoE tool support.

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies

closes #241 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

add conclusion/congratulations page

### CODE REVIEW

**Review by oindrillac — COMMENTED** [review]
/lgtm 👍

File: docs/source/conclusion.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Congratulations, you have completed the Elyra AIDevSecOps tutorial!
+
+In completing this tutorial, you have been guided through an end-to-end data science workflow aided by AICoE tools.
+
+You started with **[Meteor](http://meteor-shower-aicoe-meteor.apps.zero.massopen.cloud)** hosted by **[Operate First](https://operate-first.cloud)**. Next, you used **[JupyterHub](https://jupyter.org/hub)** to launch notebook images with Jupyter tooling and leveraged **[Project Thoth](https://thoth-station.ninja/)** bots to intelligently manage your dependencies. With your notebooks running, **[Elyra](https://github.com/elyra-ai/elyra)** JupyterLab extensions supported your AI projects with add-ons such as **[Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/)** for end to end experiments. Finally, you deploy and monitor your model using Flask.
```
Comment (MichaelClifford) [inline docs/source/conclusion.md:5]:
```suggestion
You started with either **[ Project Meteor](http://meteor-shower-aicoe-meteor.apps.zero.massopen.cloud)** or **[Operate First](https://operate-first.cloud)** to deploy your own **[JupyterHub](https://jupyter.org/hub)** instance, launch notebook images with Jupyter tooling and leverage **[Project Thoth](https://thoth-station.ninja/)** bots to intelligently manage your dependencies. With your notebooks running, **[Elyra](https://github.com/elyra-ai/elyra)** JupyterLab extensions supported your AI projects with add-ons such as **[Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/)** for end-to-end experiments. Finally, you deployed and monitored your model using Flask.
```

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

File: docs/source/conclusion.md (hunk 1 above)
Comment (chauhankaranraj) [inline docs/source/conclusion.md:5]:
```suggestion
You started with **[Meteor](http://meteor-shower-aicoe-meteor.apps.zero.massopen.cloud)** hosted by **[Operate First](https://operate-first.cloud)**. Next, you used **[JupyterHub](https://jupyter.org/hub)** to launch notebook images with Jupyter tooling and leveraged **[Project Thoth](https://thoth-station.ninja/)** bots to intelligently manage your dependencies. With your notebooks running, **[Elyra](https://github.com/elyra-ai/elyra)** JupyterLab extensions supported your AI projects with add-ons such as **[Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/)** for end to end experiments. Finally, you deployed your model as a Flask application, and set up monitoring using Prometheus and Grafana.
```

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]

**Review by chauhankaranraj — APPROVED** [review]
/lgtm :rocket:

**Pull request merged**